database/migrations/2014_12_23_130853_version_and_revision_talks.php

```diff
@@ -3,8 +3,8 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
-class VersionAndRevisionTalks extends Migration {
-
+class VersionAndRevisionTalks extends Migration
```

Comment:
it seems like @Nitpick-CI isn't picking up the `nitpick.json`.  it seems like it's not aware that a configuration was added after this pull request was originally opened.  @adamwathan am i correct  or is something else going on?